src/backend/joanie/core/api/client.py

```diff
@@ -75,7 +77,7 @@ def get_queryset(self):
                 course__accesses__user=self.request.user
             ).select_related("course", "product")
 
-        return super().get_queryset()
+        return models.CourseProductRelation.objects.filter(id=self.kwargs["id"])
```

Comment:
```suggestion
        return models.CourseProductRelation.objects.filter(
                course__accesses__user=self.request.user
            ).get(id=self.kwargs["id"])
```
@jb there's my mistake